src/main/kotlin/org/malachite/estella/process/api/ProcessExceptionHandler.kt

```diff
@@ -18,4 +19,8 @@ class ProcessExceptionHandler {
     @ExceptionHandler(InvalidStagesListException::class)
     fun handleInvalidStagesListException(ex: InvalidStagesListException) =
         ResponseEntity.badRequest().body(Message(ex.error))
+
+    @ExceptionHandler(InvalidEndDateException::class)
+    fun handleInvalidEndDateException(ex: InvalidEndDateException) =
+        ResponseEntity.badRequest().body(Message("Date you provided is before start date of process!"))
```

Comment:
na froncie widziałem że też nie pozwalałeś na wrzucanie EndDate później niż "dzisiaj", a tutaj info z błędu mówi tylko że jest wrzucone przed startDate. O co cho?

Reply:
Kurde tak szczerze nie wiem. W sensie na froncie wydawało mi się to logiczne, żeby było później, niż dzisiaj, bo nie możesz zrobić procesu, który skończy się wczoraj. Z drugiej strony ten drugi warunek też jest potrzeny. Tu wrzucę jeszcze teraz, na froncie naprawię w ES-188, bo nie do końca wiem pamiętam, co tam mam, a to jest powiązany task